VMs/metrics/README.md

```diff
@@ -137,6 +153,63 @@ $ ./7_delete_clone.sh ubuntu16.04
 # Otherwise, now you have a master VM set up you are ready to configure Jenkins.
 ```
 
+## Configuring the user on the host
+
+A user needs to be chosen on the host system that will execute the VMs. That user
+requires some configuration. The following documents an installtion with the user
+`jenkins`. The host system and user require some setup to enable use of the scripts.
+
+### Add golang `bin` directories to path
+
+The user must have both the GOLANG bin and their GOPATH bin directories in their
+path. This can be effected by adding the following to their .profile:
+
+```bash
+# set PATH so it includes golang
+if [ -d "/usr/local/go/bin" ] ; then
+	PATH="/usr/local/go/bin:$PATH"
```

Comment:
You could simplify this and the test below to use the shorthand version:

```
[ -d "/usr/local/go/bin" ] && PATH="/usr/local/go/bin:$PATH"
```

Reply:
Sure. I stole/copied the longhand from the existing default .profile I found in place on my system.